x-pack/plugins/security/server/lib/authorization/privileges/feature_privilege_builder/saved_objects_management.ts

```diff
@@ -13,6 +13,9 @@ import { BaseFeaturePrivilegeBuilder } from './feature_privilege_builder';
 
 export class FeaturePrivilegeSavedObjectsManagementBuilder extends BaseFeaturePrivilegeBuilder {
   public getActions(privilegeDefinition: FeatureKibanaPrivileges, feature: Feature): string[] {
+    if (feature.id !== 'savedObjectsManagement') {
```

Comment:
Rather ugly, but IMO, "Saved Objects Management" has always been a bit of a one-off. If we don't like this explicit feature ID check, we could alternatively add a flag to the feature interface similar to `includeInBaseRead` to indicate that the feature should `includeInSavedObjectsManagement`.

We had also discussed removing these specific type-based capabilities if/once they're no longer needed. If they're only consumed by the SOM UI, we could always convert these to traditional UI Capabilities too, via the `privileges.ui` array when we register the feature itself.

Reply:
> We had also discussed removing these specific type-based capabilities if/once they're no longer needed. If they're only consumed by the SOM UI, we could always convert these to traditional UI Capabilities too, via the privileges.ui array when we register the feature itself.

I think this is likely the proper course forward. Not doing so now to ensure we don't break @mikecote's efforts seems reasonable though. You mind adding a `//TODO` or creating an issue for us fixing this long-term?